AWSCore/Authentication/AWSCredentialsProvider.h

```diff
@@ -222,7 +222,8 @@ typedef NS_ENUM(NSInteger, AWSCognitoCredentialsProviderErrorType) {
 - (instancetype)initWithRegionType:(AWSRegionType)regionType
                      unauthRoleArn:(nullable NSString *)unauthRoleArn
                        authRoleArn:(nullable NSString *)authRoleArn
-                  identityProvider:(id<AWSCognitoCredentialsProviderHelper>)identityProvider;
+                  identityProvider:(id<AWSCognitoCredentialsProviderHelper>)identityProvider
```

Comment:
As an SDK we do not make breaking API changes. Can you overload the current method and add `sessionDuration` as a parameter in the overloaded method instead of modifying the current method?